monailabel/monailabel_HelloWorld_radiology_3dslicer.ipynb

```diff
@@ -102,7 +102,9 @@
    "source": [
     "### 1.4 Starting MONAI Label Server\n",
     "\n",
-    "The segmentation_spleen model will automatically be downloaded when starting the MONAI Label server. Network and checkpoint will be processed within the initialization steps of monailabel command."
+    "The segmentation_spleen model will automatically be downloaded when starting the MONAI Label server. Network and checkpoint will be processed within the initialization steps of monailabel command.\n",
+    "\n",
+    "**Note:** Be aware that if you're running a MONAI Label server in this notebook and you wish to close the notebook, be sure to terminate the kernel first. If you don't do that and try to start the server at a later time, you may encounter an issue where the port is still occupied from the previous session. To avoid this, always remember to terminate the kernel before closing the notebook."
```

Comment:
```
**Note:** If you are running a MONAI Label server in this notebook and plan to close the notebook, ensure that you terminate the kernel first. Failing to do so might result in an issue where the port remains occupied from the previous session when you try to restart the server later. To prevent this, always terminate the kernel before closing the notebook."
```

Reply:
Above is another version from GPT. I felt the tone is a bit more formal.

Reply:
Hi Mingxin, thanks for the quick feedback, update all related parts in the latest commit!